src/main/java/com/stripe/model/Charge.java

```diff
@@ -218,6 +219,24 @@ public void setInvoiceObject(Invoice c) {
   }
   // </editor-fold>
 
+  // <editor-fold desc="onBehalfOf">
+  public String getOnBehalfOf() {
+    return (this.onBehalfOf != null) ? this.onBehalfOf.getId() : null;
+  }
+
+  public void setOnBehalfOf(String onBehalfOfID) {
+    this.onBehalfOf = APIResource.setExpandableFieldID(onBehalfOfID, this.destination);
```

Comment:
I don't understand why `this.destination` is used here, can you clarify?

Reply:
Copy/paste fail -_- Thanks!